src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveCompletionItemProvider.cs

```diff
@@ -146,7 +146,8 @@ internal static bool IsDirectiveCompletableToken(AspNetCore.Razor.Language.Synta
         {
             return token.Kind == SyntaxKind.Identifier ||
                 // Marker symbol
-                token.Kind == SyntaxKind.Marker;
+                token.Kind == SyntaxKind.Marker ||
+                token.Kind == SyntaxKind.Keyword;
```

Comment:
Out of curiosity how did `inject` completion work before if this case was missing?

Reply:
[Not very well](https://github.com/dotnet/aspnetcore/issues/35309). Basically the fact that `in` was a keyword was hidden behind the <2 check I also changed in this PR.